tests/tidb_rowid/run.sh

```diff
@@ -17,6 +17,11 @@
 
 set -eu
 
+# disable cluster index by default
+run_sql 'set @@global.tidb_enable_clustered_index = 0' || echo "tidb does not support cluster index yet, skipped!"
```

Comment:
I think it's better to check the origin setting like https://github.com/pingcap/tidb-lightning/blob/463e7289fcbc412a49e50b307e3b33d30da7f218/tests/row-format-v2/run.sh#L8-L28

BTW, the newest version of `show variables like % tidb_enable_clustered_index%` return ON/OFF